parquet-benchmarks/src/main/java/org/apache/parquet/benchmarks/DataGenerator.java

```diff
@@ -126,14 +160,36 @@ public void cleanup()
   }
 
   public static void main(String[] args) {
-    DataGenerator generator = new DataGenerator();
+    DataGenerator generator = new DataGenerator(new Configuration());
+
     if (args.length < 1) {
-      System.err.println("Please specify a command (generate|cleanup).");
+      System.err.println("Please specify a command (generate VERSION|cleanup).");
       System.exit(1);
     }
 
     String command = args[0];
     if (command.equalsIgnoreCase("generate")) {
+      if (args.length < 2 || args.length > 3) {
+        System.err.println("Usage: generate VERSION [-randomData]\n");
+        System.err.println("Options:");
+        System.err.println("  VERSION         Use a specific parquet file version to generate data.");
+        System.err.println("                  - v1 for PARQUET_1_0 version.");
+        System.err.println("                  - v2 for PARQUET_2_0 version.");
+        System.err.println("  -randomData     This flag specifies if random data will be used to generate data.");
+        System.err.println("                  By default, only 1 random row is generated, and repeated across the file.");
+        System.exit(1);
+      }
+
+      String parquetVersion = args[1];
+
+      if (args.length == 3 && args[2].equalsIgnoreCase("-randomData")) {
```

Comment:
Seems like to generate data I *have to* specify `-randomData`, then why is that a cmd line flag at all? I tried a few command line args to see how it behaves

- `java -cp ${SCRIPT_PATH}/target/parquet-benchmarks.jar org.apache.parquet.benchmarks.DataGenerator generate`

```
Generating test data ---> This shouldn't be printed btw.
Usage: generate VERSION [-randomData]

Options:
  VERSION         Use a specific parquet file version to generate data.
                  - v1 for PARQUET_1_0 version.
                  - v2 for PARQUET_2_0 version.
  -randomData     This flag specifies if random data will be used to generate data.
                  By default, only 1 random row is generated, and repeated across the file.
```

- `java -cp ${SCRIPT_PATH}/target/parquet-benchmarks.jar org.apache.parquet.benchmarks.DataGenerator generate v2`

```
Generating test data  ---> This shouldn't be printed btw.
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 2
	at org.apache.parquet.benchmarks.DataGenerator.main(DataGenerator.java:188)
```
- `java -cp ${SCRIPT_PATH}/target/parquet-benchmarks.jar org.apache.parquet.benchmarks.DataGenerator generate v2 -randomData`

``` This one runs successfully ```